packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js

```diff
@@ -14,6 +14,8 @@ import {
   _CONVERSATION_URL_,
   _LOCUS_ID_,
   _MEETING_LINK_,
+  _PEOPLE_,
```

Comment:
You should utilize the constants from `import {hydraTypes} from '@webex/common'`

Reply:
i will change people to reference hydraTypes